src/proto/project.proto

```diff
@@ -75,6 +75,21 @@ message DeleteProjectRequest {
   optional string owner_id = 2; // The ID of the organization owning the project, empty for default org for user
 }
 
+message QueryAssetsRequest {
+  string id = 1;                    // ID of the project to request, UUIDv4, empty string uses default project
+  optional string owner_id = 2;     // The ID of the organization owning the assets, unset for default org for user
+  optional uint64 since = 3;        // The unix time to start listing assets from, unset for no restriction
```

Comment:
what unit? nanoseconds? milliseconds?

Reply:
Going with seconds as all other times in this proto are UTC seconds, I don't think we need finer resolution on this than seconds.